schema.sql

```diff
@@ -1,4 +1,5 @@
 /* Database schema to keep the structure of entire database. */
+-- HELLO HELLO HELLO
 
```

Comment:
- [ ] Please kindly update the data type of `weight_kg`, for this particular project, the appropriate data type is `decimal`. Something like so; `weight_kg decimal`

Reply:
Hi @Mihndim2020,
`numeric` and `decimal` are the same according to [PostgreSQL documentation.](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-NUMERIC-DECIMAL)

![numeric](https://user-images.githubusercontent.com/83017311/165115874-156e5f43-9fd0-4154-834a-822760b7ab7c.png)